app/models/data/returning_users.rb

```diff
@@ -19,44 +16,33 @@ def dashboard
           {
             weekly: weekly,
             monthly: monthly,
-            bimonthly: bimonthly,
             quarterly: quarterly,
           },
         ]
       end
 
     private
 
-      # @param previous [Range]
-      # @param current [Range]
+      # @Param [Range<Time>] time_range
       # @return [Integer]
-      def count(previous:, current:)
-        previous_visits = Ahoy::Visit.where(started_at: previous)
-        current_visits = Ahoy::Visit.where(started_at: current)
-
-        previous_visits.where(user_id: current_visits.pluck(:user_id)).distinct.count(:user_id)
+      def count(time_range)
```

Comment:
Without looking inside the method is it obvious what exactly is being counted?

Reply:
Is this better? [f38d173](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/749/commits/f38d1738264e7d3996999db663972aef998d1e54)